src/ontology/mondo-ingest.Makefile

```diff
@@ -345,13 +345,17 @@ documentation: j2 $(ALL_DOCS) unmapped-terms-docs mapped-deprecated-terms-docs s
 build-mondo-ingest:
 	$(MAKE) refresh-imports exclusions-all slurp-all mappings matches \
 		mapped-deprecated-terms mapping-progress-report \
-		recreate-unmapped-components sync documentation
-	$(MAKE) prepare_release
```

Comment:
This $(MAKE) caused the imports to be run twice.. No idea why, but this fixes it for now.